frontend-web/webclient/app/Project/Grant/GrantApplicationEditor.tsx

```diff
@@ -1110,24 +1107,24 @@ export const GrantApplicationEditor: (target: RequestTarget) =>
 
 interface TransferApplicationPromptProps {
     isActive: boolean;
-    username: string;
+    grantID: number;
     close(): void;
     transfer(toProjectId: string): Promise<void>;
 }
 
-function TransferApplicationPrompt({isActive, close, transfer, username}: TransferApplicationPromptProps) {
-    const [projects, fetchProjects] = useCloudAPI<FindAffiliationsResponse>(findAffiliations({page: 0, itemsPerPage: 100, username}), emptyPage);
+function TransferApplicationPrompt({isActive, close, transfer, grantID}: TransferApplicationPromptProps) {
+    const [projects, fetchProjects] = useCloudAPI<GrantsRetrieveAffiliationsResponse>(findAffiliations({page: 0, itemsPerPage: 100, grantID}), emptyPage);
 
     /* FIXME: Work-around for showing modal AND dialog. Change to hold-to-confirm button when merged with scheduling */
     const [isConfirming, setIsConfirming] = useState(false);
 
     const history = useHistory();
 
     React.useEffect(() => {
-        if (username) {
-            fetchProjects(findAffiliations({page: 0, itemsPerPage: 100, username}))
+        if (grantID) {
+            fetchProjects(findAffiliations({page: 0, itemsPerPage: 100, grantID}))
```

Comment:
You're positive that `grantID` is the correct thing to submit?

Reply:
There have been a change to the api. It now takes the applicationID instead of the username of the applicant.